hack/e2e-test.sh

```diff
@@ -38,7 +38,10 @@ function startup {
 }
 
 function kind_load {
-    $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
+    if [ $USE_EXISTING_CLUSTER == 'false' ]
```

Comment:
```suggestion
    if [ $CREATE_KIND_CLUSTER == 'true' ]
```

Maybe this is more readable